### PR TITLE
Update the ruby pubsub sample to work in App Engine standard env as well

### PR DESCRIPTION
**Pull request by darrencarlton**

### CODE REVIEW

File: appengine/pubsub/app.yaml [hunk 1]
```diff
@@ -16,10 +16,10 @@ runtime: ruby
 env: flex
 entrypoint: bundle exec ruby app.rb
 
-# [START gae_flex_pubsub_yaml]
+# [START gae_pubsub_yaml]
```
Comment (averikitsch) [inline appengine/pubsub/app.yaml]: Do you want to change this region_tag? You'll have to update all the docs.

File: appengine/pubsub/app.standard.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Google, Inc
```
Comment (averikitsch) [inline appengine/pubsub/app.standard.yaml]:
```suggestion
# Copyright 2019 Google LLC
```

File: appengine/pubsub/app.standard.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+runtime: ruby25
+entrypoint: bundle exec ruby app.rb
+
```
Comment (averikitsch) [inline appengine/pubsub/app.standard.yaml]: Does this need an instance class? https://github.com/GoogleCloudPlatform/ruby-docs-samples/blob/master/appengine/standard-hello_world/app.yaml#L18

File: appengine/pubsub/app.yaml (hunk 1 above)
Comment (darrencarlton) [reply appengine/pubsub/app.yaml]: Yes, having one tag for both flex and standard simplifies the doc tagging.

File: appengine/pubsub/app.standard.yaml (hunk 3 above)
Comment (darrencarlton) [reply appengine/pubsub/app.standard.yaml]: Sure, let's add F2 as in the other sample, It isn't necessary, but seems like a reasonable setting.
I'll test before committing the PR

Comment (darrencarlton) [reply appengine/pubsub/app.standard.yaml]: app deploys and runs with instance_class: F2

**Comment by darrencarlton** [comment]
Hi Averi:

I don't have permission to merge the pull request. Can you do the merge?

**Comment by darrencarlton** [comment]
Thanks, Tyler

**Pull request merged**